src/client/interactive.ts

```diff
@@ -0,0 +1,7 @@
+const body = document.querySelector("body");
+const userAgent = navigator.userAgent || navigator.vendor;
+
+if (/android/i.test(userAgent)) {
+    body?.classList.add("android");
+    body?.classList.remove("ios");
+}
```

Comment:
Are we likely to run up against Chrome's plans to [phase out](https://wicg.github.io/ua-client-hints/) UA strings?

Reply:
Possibly, this is for backwards compatibility but if we update our styles and classnames to match what dotcom-rendering do, we shouldn't need this.